AlphaWallet/Core/Initializers/MigrationInitializer.swift

```diff
@@ -60,5 +60,37 @@ class MigrationInitializer: Initializer {
                 }
             }
         }
+        config.migrationBlock = { migration, oldSchemaVersion in
```

Comment:
I understand this is a work in progress, but in the final commit, can you change the migration version to be just +1 (i.e. 6 instead of 8)?

Also, migration needs to take care of moving contracts in `HiddenContracts` over to `TokenObject` and marked as hidden